drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -41,5 +48,29 @@ def self.sort_by_option(option)
     end
   end
 
-  use_enum :relationships_to_hoh_enum_map, ::HUD.relationships_to_hoh
+  def save_in_progress
+    self.wip = Hmis::Wip.find_or_create_by(
+      {
+        enrollment_id: id,
+        project_id: project_id,
+        client_id: personal_id,
```

Comment:
We want the `client.id` here not the personal ID! thanks

Reply:
Fixed